src/notification/user_notification.py

```diff
@@ -40,10 +40,20 @@ def generate_notification(report):
         result["data"]["attributes"]["custom"]["scanned_at"] = \
             strftime("%a, %d %B %Y %T GMT", gmtime())
         vulnerable_deps = result["data"]["attributes"]["custom"]["vulnerable_deps"]
+        del result["data"]["attributes"]["custom"]["vulnerable_deps"]
         total_cve_count = 0
+        transitive_updates = list()
+        direct_updates = list()
 
         for deps in vulnerable_deps:
+            is_transitive = deps.get("is_transitive", None)
+            if is_transitive:
+                transitive_updates.append(deps)
+            else:
+                direct_updates.append(deps)
             total_cve_count += int(deps['cve_count'])
         result["data"]["attributes"]["custom"]["cve_count"] = total_cve_count
-
+        result["data"]["attributes"]["custom"]["transitive_updates"] = transitive_updates
+        result["data"]["attributes"]["custom"]["direct_updates"] = direct_updates
+        print("********************PAYLOAD****************************", result)
```

Comment:
@GeetikaBatra can this message be more meaningful? Also, I am assuming that `print` is being used temporarily and will be removed after a thorough testing of notification is done on staging, isn't it?

Reply:
@samuzzal-choudhury Yep this is temporary, I will be removing it as soon as I know that this piece of code works fine. Do I still need to make this message more meaningful?

Reply:
@GeetikaBatra yes, a log with something with more meaningful.

Reply:
@samuzzal-choudhury, does log statements appear in the terminal. I have observed that they are missed in staging logs, that's why I am using print statements.

Reply:
Better option will be to introduce FLASK_LOGGING_LEVEL env variable and replace the print with logger